buildSrc/src/main/java/quilt/internal/task/MappingsTask.java

```diff
@@ -0,0 +1,9 @@
+package quilt.internal.task;
+
+import org.gradle.api.Task;
+import quilt.internal.plugin.QuiltMappingsPlugin;
+
+/**
+ * All tasks added by {@link QuiltMappingsPlugin QuiltMappingsPlugin} share this type.
+ */
+public interface MappingsTask extends Task { }
```

Comment:
Do we need this anymore or at all?

Reply:
Not really, I kept it in case we wanted to add commonly injected services or broad convenience methods similar to `MappingsProjectPlugin`, or in case we want to identify plugin tasks by type.

But since the PR is basically ready and this isn't used I think that's unlikely.

If we remove it we can also remove `DefaultMappingsTask` and just set groups in each task.

Reply:
eliminated both